### DISCUSSION TITLE
Bayesian Birds

### DISCUSSION THREAD

**Discussion by incandescental** [mcguirepr89/BirdNET-Pi#725] [Ideas]
I've had my BirdNET-Pi set up for a couple of weeks and am really enjoying it - well done to everyone who's been involved in making it happen. 

I've had apprise notifications set up for infrequent detections but find I'm mostly just getting the false positives - including my saxophone being misidentified as an eagle owl (and everyone knows they hate jazz). I've seen the options for allowed/ exclusion lists but I was wondering if anyone was exploring a more probabilistic approach based on data from other sources? 

The aim of the apprise notification is to alert of anything unusual so you want 'unlikely but possible' to still trigger given a sufficiently high confidence recording from BirdNET. Conversely, a species that has never been seen on this continent needs an almost zero weighting.

Would it be possible to incorporate data from other bird atlas type projects (i.e. the BTO or EBCC) along with the lat/lon of the device to add some sort of weighting rather than a on/off species list?

**Comment by incandescental** [comment]
Ahh, ignore that - it sounds like this might already be covered in the new model discussion

**Reply by ehpersonal38** [reply]
Yep, the new model offers exactly this functionality :) It's available to everyone now by updating via Tools -> System Controls -> then changing your settings in Tools -> Settings.
